po/common/common.pot

```diff
@@ -3330,8 +3330,8 @@ msgstr "Import a product data file"
 
 # "product data" in this sentence means data for many products, not just one product
 msgctxt "import_data_file_description"
-msgid "Upload a spreadsheet file (Excel file or a tab separated UTF-8 encoded CSV file) with product data."
-msgstr "Upload a spreadsheet file (Excel file or a tab separated UTF-8 encoded CSV file) with product data."
+msgid "Upload a spreadsheet file (Excel file or a comma or tab separated UTF-8 encoded CSV file) with product data."
```

Comment:
```suggestion
msgid "Upload a spreadsheet file (Excel file or a comma/tab separated UTF-8 encoded CSV file) with product data."
```

Reply:
I'm not sure comma/tab is more clear, I'd prefer to keep the or.